kong.conf.default

```diff
@@ -269,6 +269,14 @@
                                  # Similarly to `error_template_html`, the template
                                  # is required to contain one single `%s` placeholder for
                                  # the error message.
+
+#worker_events_max_payload = 65535  # Specifies the max payload size the
```

Comment:
Do we still need to document the change?
@hbagdi @dndx

Reply:
Removed.